client/templates/orderDetails/status/orderReadyToShip/orderReadyToShip.js

```diff
@@ -3,6 +3,14 @@ Template.orderReadyToShip.events({
     event.preventDefault();
     let orderId = this._id;
     let userId = Meteor.userId();
-    Meteor.call('advancedFulfillment/updateOrderWorkflow', orderId, userId, 'orderReadyToShip');
+    let noRentals = _.every(this.advancedFulfillment.items, function (afItem) {
```

Comment:
a name like `orderHasNoRentalItems` might be more clear as to what this is doing.